framework/Optimizers/GradientDescent.py

```diff
@@ -304,6 +304,8 @@ def _useRealization(self, info, rlz, optVal):
       gradMag, gradVersor, _ = self._gradientInstance.evaluate(opt,
                                                                grads, gradInfos,
                                                                self._objectiveVar)
+
+      print("This is gradMag, gradVersor")
```

Comment:
All of these debug prints should be removed.

Reply:
Not a problem